src/librustdoc/html/markdown.rs

```diff
@@ -645,7 +653,14 @@ impl LangString {
                     seen_rust_tags = seen_other_tags == false;
                 }
                 "no_run" => { data.no_run = true; seen_rust_tags = !seen_other_tags; }
-                "ignore" => { data.ignore = true; seen_rust_tags = !seen_other_tags; }
+                "ignore" => {
+                    data.ignore = Ignore::All;
+                    seen_rust_tags = !seen_other_tags;
+                }
+                x if x.starts_with("ignore-") => {
```

Comment:
Is this insta-stable?

Reply:
I do not know what that means

Reply:
I think you're adding a feature to the language that is immediately available to users without a feature gate.

Reply:
Oh, I think that is how I wrote it. I didn't even consider having the new behavior as an option.

Reply:
The last similar change was #54350 which looks like it just landed without stability concerns.